File: src/main/java/com/github/wgzhao/dbquery/repo/QueryConfigRepo.java
package com.github.wgzhao.dbquery.repo;

import com.github.wgzhao.dbquery.entities.QueryConfig;
import org.springframework.data.jpa.repository.JpaRepository;

public interface QueryConfigRepo extends JpaRepository<QueryConfig, String> {
    boolean existsByDataSource(String id);
}

File: src/main/kotlin/com/github/wgzhao/dbquery/repo/QueryConfigRepo.kt
package com.github.wgzhao.dbquery.repo

import com.github.wgzhao.dbquery.entities.QueryConfig
import org.springframework.data.jpa.repository.JpaRepository

interface QueryConfigRepo : JpaRepository<QueryConfig?, String?> {
    fun existsByDataSource(id: String?): Boolean
}
